screenplay.txt

```diff
@@ -121,6 +121,8 @@ Finally, everyone shouted... "We had an awesome day.... Amazing days ahead..."
 
 THE END
 
+Changed by William McCann
+
```

Comment:
This looks good. Perhaps you could add some narrative, as per Daniel's suggestion?